internal/querycoord/task_scheduler.go

```diff
@@ -776,21 +785,26 @@ func (scheduler *TaskScheduler) waitActivateTaskDone(wg *sync.WaitGroup, t task,
 	}
 
 	redoFunc2 := func(err error) {
-		if t.isValid() {
-			if !t.isRetryable() {
-				log.Error("waitActivateTaskDone: activate task failed after retry",
-					zap.Int64("taskID", t.getTaskID()),
-					zap.Int64("triggerTaskID", triggerTask.getTaskID()))
-				triggerTask.setResultInfo(err)
-				return
-			}
-			log.Debug("waitActivateTaskDone: retry the active task",
+		if !t.isValid() {
```

Comment:
Task use isValid interface to check precondition, the function name is confusing.